.github/workflows/plugins-ci-postgres.yml

```diff
@@ -98,8 +103,8 @@ jobs:
       contents: read
     strategy:
       matrix:
-        node-version: [18, 20, 21]
-        os: [ubuntu-latest]
+        node-version: ${{ fromJson(inputs.node-versions) }}
+        os: [macos-latest, ubuntu-latest, windows-latest]
```

Comment:
The workflows that were only using Ubuntu should stay using only Ubuntu, as you cannot use MacOS or Windows with services.

See https://docs.github.com/en/actions/using-containerized-services/about-service-containers#about-service-containers

Reply:
Oh yes, sorry for that

Will fix